src/OpenTelemetry.Instrumentation.AWS/Implementation/AWSServiceHelper.cs

```diff
@@ -7,16 +7,18 @@ namespace OpenTelemetry.Instrumentation.AWS.Implementation;
 
 internal class AWSServiceHelper
 {
-    internal static IReadOnlyDictionary<string, string> ServiceParameterMap = new Dictionary<string, string>()
+    internal static IReadOnlyDictionary<string, List<string>> ServiceParameterMap = new Dictionary<string, List<string>>()
```

Comment:
Curious - why was it necessary to change from `Dictionary<string, string>` to `Dictionary<string, List<string>`?

Looks like every key-value pair in the dictionary only has a single value.

Reply:
This change was not necessary for the services supported in this commit, but in preparation for two additional services - Bedrock Agent and Bedrock Agent Runtime - that will be supported and each contain multiple request parameters of interest. #1962 includes the same change in order to update support for SQS.